File: ThaiTuanERP2025.Server/ThaiTuanERP2025.Application/Finance/DTOs/TaxDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using ThaiTuanERP2025.Domain.Finance.Enums;

namespace ThaiTuanERP2025.Application.Finance.DTOs
{
	public record TaxDto(
		Guid Id, string PolicyName, decimal Rate, TaxBroadType TaxBroadType, ConsumptionSubType? ConsumptionSubType,
		Guid PostingLedgerAccountId, string PostingLedgerAccountNumber, string PostingLedgerAccountName, 
		string? Description, bool IsActive
	);
}

File: ThaiTuanERP2025.Client/src/app/pages/finance/models/tax.dto.ts
export interface TaxDto {
      policyName: string;
      rate: number;
      postingLedgerAccountId: string;
      isActive: boolean;
      description?: string | null;
}

export interface TaxRequestDto extends TaxDto {
      id: string;
      postingLedgerAccountNumber?: string;
      postingLedgerAccountName?: string;
}